src/main/java/core/basesyntax/model/User.java

```diff
@@ -8,6 +8,13 @@ public class User {
     private String password;
     private Integer age;
 
+    public User(Long id, String login, String password, Integer age) {
+        this.id = id;
+        this.login = login;
+        this.password = password;
+        this.age = age;
+    }
```

Comment:
I think that we mustn't delete getters and setters in this class. We could use these for more informative messages